docs/content/language-reference.md

```diff
@@ -159,7 +159,159 @@ The following table shows examples of how ``glob.match`` works:
 | <span class="opa-keep-it-together">``output := yaml.marshal(x)``</span> | ``output`` is ``x`` serialized to a YAML string |
 | <span class="opa-keep-it-together">``output := yaml.unmarshal(string)``</span> | ``output`` is ``string`` deserialized to a term from YAML encoded string |
 
-### Tokens
+### Token Signing
+
+OPA provides two builtins that implement JSON Web Signature [RFC7515](https://tools.ietf.org/html/rfc7515) functionality. 
+
+``io.jwt.encode_sign_raw()`` takes three JSON Objects (strings) as parameters and returns their JWS Compact Serialization. This builtin should be used by those that want maximum control over the signing and serialization procedure. It is important to remember that StringOrURI values are compared as case-sensitive strings with no transformations or canonicalizations applied. Therefore, line breaks and whitespaces are significant.
```

Comment:
Nit: this function takes JSON strings not JSON objects.

Reply:
According to the RFC it takes JSON Objects: https://tools.ietf.org/html/rfc7159#page-6

If somebody passes a string for JWK it will not work.  JSON strings according to the RFC as much like C strings.: https://tools.ietf.org/html/rfc7159#page-8.

Reply:
The language-reference.md document describes built-in functions defined in the Rego language. As defined, the `io.jwt.encode_sign_raw` built-in function takes three STRING parameters.

Reply:
Done